tests/e2e/restic_helpers.go

```diff
@@ -86,23 +110,23 @@ func areResticPodsRunning() wait.ConditionFunc {
 	}
 }
 
-func waitForResticPods() error {
+func waitForResticPods(namespace string) error {
 	// poll pod every 5 secs for 2 mins until it's running or timeout occurs
-	return wait.PollImmediate(time.Second*5, time.Minute*2, areResticPodsRunning())
+	return wait.PollImmediate(time.Second*5, time.Minute*2, areResticPodsRunning(namespace))
```

Comment:
Gomega provides [Eventually](https://onsi.github.io/gomega/#eventually) block out of the box to poll for these kinds of situations. We might be able to use this here and remove this function altogether :thinking:. Maybe call `areResticPodsRunning` from an `Eventually` block. Ignore if this is implemented for a specific reason.

Reply:
Good suggestion! I was wondering if there would be a callback for this, eventually fits the case. I will try it out.